logchange-maven-plugin/src/test/java/dev/logchange/maven_plugin/mojo/release/ReleaseVersionMojoIT.java

```diff
@@ -92,4 +90,33 @@ void releaseWithVersionInPomXmlWithoutSnapshotOrRc(MavenExecutionResult result)
         assertThat(versionSummary).exists();
     }
 
+    @MavenGoal({"${project.groupId}:${project.artifactId}:${project.version}:release"})
+    @MavenOption("-DchangesXml")
+    @MavenOption("-DoutputFileXml=\"TestChanges.xml\"")
+    @MavenTest
+    @DisplayName("Project with pom.xml with version set to 1.3.7, generating Changes.xml with custom name")
+    void releaseWithVersionInPomXmlGeneratingChangesXml(MavenExecutionResult result) {
+        assertThat(result).isSuccessful()
+                .project()
+                .has("changelog")
+                .has("changelog/unreleased")
+                .has("changelog/v1.3.7");
+
+
+        File gitKeep = new File(result.getMavenProjectResult().getTargetProjectDirectory(), "changelog/unreleased/.gitkeep");
+        File changelog = new File(result.getMavenProjectResult().getTargetProjectDirectory(), "CHANGELOG.md");
+        File taskMovedToRelease = new File(result.getMavenProjectResult().getTargetProjectDirectory(), "changelog/v1.3.7/task.yml");
+        File releaseDateFile = new File(result.getMavenProjectResult().getTargetProjectDirectory(), "changelog/v1.3.7/release-date.txt");
+        File versionSummary = new File(result.getMavenProjectResult().getTargetProjectDirectory(), "changelog/v1.3.7/version-summary.md");
+        File changesXmlFIle = new File(result.getMavenProjectResult().getTargetProjectDirectory(), "changelog/v1.3.7/version-summary.md");
```

Comment:
i think its not correct and it should be: 

File changesXmlFIle = new File(result.getMavenProjectResult().getTargetProjectDirectory(), "TestChanges.xml");

@mzaninovic555